src/services/fileStorage/proxy-service.js

```diff
@@ -557,21 +572,28 @@ const renameService = {
 
 		return canWrite(userId, _id)
 			.then(() => FileModel.findOne({ _id }).exec())
-			.then((directory) => {
-				if (!directory) return Promise.reject(new NotFound('The given directory/file was not found!'));
+			.then((obj) => {
+				if (!obj) return Promise.reject(new NotFound('The given directory/file was not found!'));
 				return FileModel.update({ _id }, { name: newName }).exec();
```

Comment:
$set missing? or patch instead of update?

Reply:
`$set` is not needed necessarily: https://mongoosejs.com/docs/api/model.html#model_Model.update